src/vm/gdbjit.cpp

```diff
@@ -371,7 +461,7 @@ GetDebugInfoFromPDB(MethodDesc* MethodDescPtr, SymbolsInfo** symInfo, unsigned i
     StackScratchBuffer scratch;
     const char* szModName = modName.GetUTF8(scratch);
 
-    MethodDebugInfo* methodDebugInfo = new (nothrow) MethodDebugInfo();
+    NewHolder<MethodDebugInfo> methodDebugInfo = new (nothrow) MethodDebugInfo();
```

Comment:
This is yet another struct that can be allocated on stack instead of using holder.